### DISCUSSION TITLE
Or, how to headlessui in vitepress

### DISCUSSION THREAD

**Discussion by zghbyslzf** [tailwindlabs/headlessui#2144] [Help]
I want to use the headless ui to re write documents in vitepress, but I find no css

I want the css style to appear

[demo.zip](https://github.com/tailwindlabs/headlessui/files/10344258/demo.zip)

**Comment by twittkopf** [comment]
Hey! Headless UI does not contain any styling, just the logic and behaviour to build accessible UI components – hence the term headless. You are free to style the components yourself by supplying your own CSS.

**Reply by zghbyslzf** [reply]
> Hey! Headless UI does not contain any styling, just the logic and behaviour to build accessible UI components – hence the term headless. You are free to style the components yourself by supplying your own CSS.

I know, but I want to write styles in the md document, but I can't. There is a demo in the demo

**Comment by RobinMalfait** [comment]
You have to install Tailwind CSS. Your package.json doesn't have Tailwind CSS as dependency and there is also no `tailwind.config.js` file.

**Reply by zghbyslzf** [reply]
> You have to install Tailwind CSS. Your package.json doesn't have Tailwind CSS as dependency and there is also no `tailwind.config.js` file.

How to install Tailwind CSS in vitepress
Only vite in the document
My own installation failed
